evacuees/src/API/EMBC.Tests.Integration.Registrants.API/Evacuations/EvacuationTests.cs

```diff
@@ -18,6 +19,57 @@ public class EvacuationTests : WebAppTestBase
         private readonly IEvacuationManager evacuationManager;
         private readonly IListsRepository listsRepository;
 
+        // Constants
+        const string TestUserId = "CHRIS-TEST";
```

Comment:
Can you convert this to an evacuation file instance instead of consts? We can later refactor the assertions to a method to compare 2 files.

Reply:
Done.